_releaser/cloudfront-lambda-redirects.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+exports.handler = (event, context, callback) => {
+    //console.log("event", JSON.stringify(event));
+    const request = event.Records[0].cf.request;
+
+    const redirects = JSON.parse(`{{.RedirectsJSON}}`);
+    for (let key in redirects) {
+        if (key !== request.uri) {
+            continue;
+        }
+        console.log(`redirect: ${request.uri} to ${redirects[key]}`);
```

Comment:
Where do these logs end up? Wondering;

- how noisy these will be (do these log account to some usage-data, or are we not charged for them?)
- should these be `console.debug` or `console.info` (or anything along those lines?)

Reply:
This can be used if CloudWatch is enabled but don't think it's enabled atm. It's was mainly for debug purpose so I think we can comment that.

Reply:
if it's not affecting the deployment, then yeah, we could either let them as-is, or (just to be sure) comment them indeed.